### ISSUE TITLE
[Truffle] Unable to sprintf bignum

### ISSUE THREAD

**Issue by bjfish** [jruby/jruby#2949]
Actual
```
$jt print '"%.32x" % 85060308944708794891899627827609206784'
core:/core/string.rb:12:in `sprintf': Truffle doesn't have a case for the org.jruby.truffle.pack.nodes.type.ToIntegerNodeGen node with values of type  Bignum(org.jruby.truffle.runtime.core.RubyBignum) (TypeError)
	from core:/core/string.rb:12:in `%'
	from -e:1:in `<main>'
```
Expected
```
ruby -e 'puts "%.32x" % 85060308944708794891899627827609206784'
3ffe0505000200000000000000000000
```

**Comment by bjfish** [comment]
This is issue was found in ipaddr stdlib.

**Comment by chrisseaton** [comment]
Fixed in 5c1ee15b4c6c0a5314a537d8b3ab7d04f73b8d96.
